.mergify.yml

```diff
@@ -39,6 +41,35 @@ pull_request_rules:
         - check-success=Quality Gate
         - -files~=^(?!docs/|logo/)
 
+  - name: requires manual merge
+    conditions:
+      - files=.mergify.yml
+      - -title~=(?i)wip
+      - -label~=(🚧 pr/do-not-merge|⚠️ pr/review-mutation|⚠️ mergify/review-config)
+      - -merged
+      - -closed
+      - "#approved-reviews-by>=1"
+      - "#changes-requested-reviews-by=0"
+      - "#review-threads-unresolved=0"
+      - -approved-reviews-by~=author
+      - check-success=Validate PR title
+      - check-success=Quality Gate
+      - or:
+        -  base=main
+        -  base=dev
+      - or:
+        - updated-at<=2023-07-13T07:00:00Z
+        - base=dev
+    actions:
+      comment:
+        message: Thank you for contributing! Your pull request contains megify configuration changes and needs manual merge from a maintainer (be sure to [allow changes to be pushed to your fork](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)).
+      label:
+        add:
+          - ⚠️ mergify/review-config
+      request_reviews:
+        teams:
+          - "@winglang/maintainers"
```

Comment:
Due to our current CODEOWNERS, they will always be reviewers on a PR. Maybe add it to the comment body so it's an `@mention` instead.

Reply:
Yes but it is right after the pr is opened, right? This one is for when all conditions are satisfied and it is ready to merge. Add a label and then notify the reviewers. Doesn't that make sense?

Reply:
Sure, I was just thinking an `@mention` would be more noticeable (e.g. I get push notifications on my phone for mentions, but not requested reviews).

I'm totally fine with just a requested review though 👍

Reply:
Nah actually I thought about this and my original plan was to mention you specifically (since you work with the mergify configs most) and then I found there's an action to request reviews in mergify and went with that. This is because I didn't know we could mention teams itself with @mention. Just checked and found this is possible so definitely the more straightforward and noticeable method. I'll change it to that.

Reply:
Seems like there are some quirks regarding team mentions so I will stick to the current config for now.